### PR TITLE
CDS-45178-Create services and environments at account and org level

### PR DESCRIPTION
**Pull request by roshnisarangadharan**
add service and env from account and org level

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by thisrohangupta — APPROVED** [review]
This looks better thanks!

File: docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md [hunk 1]
```diff
@@ -10,106 +10,557 @@ helpdocs_is_published: true
...
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+```mdx-code-block
+<Tabs>
+  <TabItem value="Harness UI" label="Harness UI">
+```
+To create a service at an account and organization level, select **Services** in **Organization Resources**.
+
```
Comment (StrawberryPad) [inline docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md:38]: it's good practice to put the user clicks in the order they do it **Organization Resources > Services**. So here, maybe:
To create a service at an account and (_should this be **or**?_) organization level, select **Organization Resources** and then **Services**.

Or to be minimal, you can say "go to **Organization Resources > Services**"

File: docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md [hunk 2]
```diff
@@ -10,106 +10,557 @@ helpdocs_is_published: true
...
+```
+</details>
+
+```mdx-code-block
+  </TabItem>
+  <TabItem value="API" label="API">
+```
+For information about creating a service API, go to [create a service](https://apidocs.harness.io/tag/Services#operation/createServiceV2).
+
+The `orgIdentifier` and `projectIdentifier` field definitions are optional, and depend on where you want to create the service. For example, if you create a service at an account level, you will not need org or project identifiers in the post api call payload.
```
Comment (StrawberryPad) [inline docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md:153]: api > API

File: docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md [hunk 3]
```diff
@@ -10,106 +10,557 @@ helpdocs_is_published: true
...
+
+To create a service from outside of a pipeline, you use **Services** in the navigation pane.
 
 ![](./static/services-and-environments-overview-09.png)
 
-To create a service from inside of a Pipeline, select **New Service** in the **Services** tab of a new CD stage.
+### Creating services inside a pipeline
 
-![](./static/services-and-environments-overview-10.png)
+To create a service from inside of a pipeline, select **New Service** in the **Services** tab of a new CD stage.
```
Comment (StrawberryPad) [inline docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md:238]: Ditto about the order of clicks

File: docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md [hunk 4]
```diff
@@ -10,106 +10,557 @@ helpdocs_is_published: true
...
+![](./static/services-and-environments-overview-10.png)
 
 Once the service and its service definition are saved, you can select it in any pipeline.
 
 When you select the service in a pipeline, you can select **Edit Service** to edit its **Service Definition**.
 
-### Creating and using environments
-
-Environments can be created from inside or outside of a pipeline.
+When you create the new service you define its **Service Definition**. For example, Kubernetes **Service Definition** with a Kubernetes manifest and Docker artifact.
```
Comment (StrawberryPad) [inline docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md:246]: For example, **a** Kubernetes...

File: docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md [hunk 5]
```diff
@@ -10,106 +10,557 @@ helpdocs_is_published: true
...
-  * For example, you might have a QA Environment with separate Kubernetes clusters (Infrastructure Definitions) for each Service you want to test.
-  * You can add multiple Infrastructure Definitions to a single Environment and select one when you add the Environment to a stage.
-* **Configuration:** the default Environment configuration, including variables, manifests, specifications, and config files that will be used every time the Environment is used in a stage.
-* **Service Overrides:** override specific Services. You select a Service and define what will be overridden whenever that Service is deployed to this Environment.
-* **GitOps Clusters:** adding Harness GitOps Clusters to an Environment lets you select them as the deployment target in stages. For more information on Harness GitOps, see [Harness GitOps Basics](../../cd-gitops/harness-git-ops-basics.md).
+  * Infrastructure definitions are the actual clusters, hosts, etc., where Harness deploys a service.
+  * For example, you might have a QA environment with separate Kubernetes clusters (infrastructure definitions) for each service you want to test.
```
Comment (StrawberryPad) [inline docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md:262]: This "for example" seems like a part of the above bullet, not a separate bullet.

File: docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md [hunk 6]
```diff
@@ -10,106 +10,557 @@ helpdocs_is_published: true
...
+  * You can add multiple infrastructure definitions to a single environment and select one when you add the environment to a stage.
```
Comment (StrawberryPad) [inline docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md:263]: "select one" - **one** refers to the previous noun (environment)... but I think **one** means an infrastructure definition here. For clarity/grammar, substitute "infrastructure definition" for **one**.

**Review by StrawberryPad — APPROVED** [review]

File: docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md [hunk 7]
```diff
@@ -10,106 +10,557 @@ helpdocs_is_published: true
...
-Services can be created from inside or outside of a Pipeline.
-
-To create a Service from outside of a pipeline, you use **Services** in the navigation pane.
+## Services
+
+Services represent your microservices and other workloads. Each service contains a **Service Definition** that defines your deployment artifacts, manifests or specifications, configuration files, and service-specific variables.
+
+You can create services from: 
+* Account level
+* Organization level
```
[the remark this answers is not in the document]
Comment (roshnisarangadharan) [reply docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md:21]: Done

File: docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md (hunk 1 above)
Comment (roshnisarangadharan) [reply docs/continuous-delivery/onboard-cd/cd-concepts/services-and-environments-overview.md:38]: Done

**Review by SudheendraKatte — APPROVED** [review]
Approving to unblock @roshnisarangadharan

**Pull request merged**